snykTask/src/task-lib.ts

```diff
@@ -183,3 +193,20 @@ function hasMatchingVulnerabilities(project: any, thresholdOrdinal: number) {
   }
   return false;
 }
+
+// finds code issues levels mapping to severity of matching or higher than threshold
+function hasMatchingCodeIssues(results: any, thresholdOrdinal: number) {
+  for (const issue of results) {
+    if (
+      getSeverityOrdinal(codeSeverityMap[issue['level']]) >= thresholdOrdinal
+    ) {
```

Comment:
nitpick(non-blocking): I don't like the formatting - the `(` `)` should be on the same line as the condition. Did prettier/linting complain?

Reply:
This is auto-formatted by Prettier. :)